### PR TITLE
Bug: fixed creating multiple modals affecting performance and backdrop color

### PR DESCRIPTION
**Pull request by ishinan**
fixed creating multiple modals affecting performance and rendering backdrop color.
Cleaned some codes which were not required.

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -112,40 +110,35 @@ fetchMaterials = () => {
...
-                      console.log(facility_data);
-                      this.setState({facilityDetails: facility_data.results.result[facility_id]});
-                    }).then((data) => {
-                      const myPromise = new Promise(function(resolve, reject) {
-                         resolve();
-                      });
-                      return myPromise
-                    });
-  return myPromise
+  fetch(url, {})
```
Comment (sudo-joseph) [inline client/src/App.js:140]: so much cleaner. great job

**Review by sudo-joseph — DISMISSED** [review]
Great work. Thanks for fixing the modal.

File: client/src/App.js (hunk 1 above)
Comment (ishinan) [reply client/src/App.js:140]: :)

**Comment by ishinan** [comment]
@sudo-joseph  Can you review it again? There was conflicts I needed to resolve.

**Review by sudo-joseph — DISMISSED** [review]
Thanks for fixing merge conflicts.

**Pull request merged**